convert/jenkinsjson/convert.go

```diff
@@ -662,6 +661,10 @@ func collectStepsWithID(currentNode jenkinsjson.Node, stepWithIDList *[]StepWith
 			index++
 		}
 
+	case "allure":
+		*stepWithIDList = append(*stepWithIDList, StepWithID{Step: jenkinsjson.ConvertAllureReport(currentNode), ID: id})
+		*stepWithIDList = append(*stepWithIDList, StepWithID{Step: jenkinsjson.Converts3UploadStep(currentNode), ID: id})
+
```

Comment:
since allure report generation and upload steps are part of the same functionality, we can combine these both in allure.go leading to a single call from this `case "allure":` block

you could create a new func `ConvertAllureSteps` in allure.go that calls both allure report generation and upload steps and call this new func from the case block.

Reply:
@Ompragash, Addressed the above comment, could you please review it once and confirm?